### PR TITLE
React App - add logo select

### PR DESCRIPTION
**Pull request by adamnitti**
## overall purpose
Adds logo select option for signature creation

## screenshot
Screen Shot 2022-12-29 at 11 21 26 AM
Screen Shot 2022-12-29 at 11 21 55 AM
Screen Shot 2022-12-29 at 11 22 12 AM
Screen Shot 2022-12-29 at 11 22 29 AM

### CODE REVIEW

File: signature-react-app/src/App.tsx [hunk 1]
```diff
@@ -81,6 +102,17 @@ function App() {
     }));
   };
 
+  const handleChangeLogo = (
```
Comment (adamnitti) [inline signature-react-app/src/App.tsx:117]: logo change handler

File: signature-react-app/src/App.tsx [hunk 2]
```diff
@@ -150,71 +183,102 @@ function App() {
...
+                    </MenuItem>
+                    <MenuItem value={LOGOS["ata-cpa-advisors"]}>
+                      ATA Cpa Advisors
+                    </MenuItem>
+                    <MenuItem value={LOGOS["ata-employment-solutions"]}>
+                      ATA Employment Solutions
+                    </MenuItem>
+                  </Select>
+                </FormControl>
+              </Box>
```
Comment (adamnitti) [inline signature-react-app/src/App.tsx]: adding MUI select component

File: signature-react-app/src/Signature.tsx [hunk 3]
```diff
@@ -25,7 +24,7 @@ const Signature = (props: PhotoSignatureProps) => {
                   <td>
                     <img
                       style={{ width: "120px", height: "90px" }}
-                      src={Logo}
+                      src={props.logo}
```
Comment (adamnitti) [inline signature-react-app/src/Signature.tsx:27]: updating displayed/selected logo, passed in via props

File: signature-react-app/src/App.tsx [hunk 4]
```diff
@@ -150,71 +183,102 @@ function App() {
...
+                    sx={{
+                      width: 250,
+                      height: 50,
+                      marginLeft: 0.7,
+                    }}
+                    value={state.logo}
+                    placeholder="Select a Logo"
+                    name="logo"
+                    onChange={handleChangeLogo}
+                  >
```
Comment (chancesmith) [inline signature-react-app/src/App.tsx]: move inline styles to useStyles (like class names)

**Review by chancesmith — APPROVED** [review]
just a few styles to relocate, but this is great!

**Pull request merged**